benchmark.cfg.example

```diff
@@ -20,7 +20,7 @@ query_levels=[1, 5,10,15,20]
 threads=8
 mode=benchmark
 os=linux
-server_host=127.0.0.1
+server_host=TFB-server
```

Comment:
If it is the case, those frameworks can still do the `sed` stuff they're doing now and pull in the ip from the `/etc/hosts` file.